### DISCUSSION TITLE
辨識木瓜熟度的AI應用

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS4719-C/Python-Community-GS4719-C#140] [科技新知]
新聞網址：https://news.sina.com.tw/article/20211029/40410482.html
題目網址：https://aigo.org.tw/zh-tw/competitions/details/404

用影像辨識加上聊天機器人，可以做一個辨識木瓜熟度的AI應用

AI做的事情大致上分成兩類：回歸、分類

辨識木瓜熟度的應用屬於分類的影用

**Comment by enoch-lightning** [comment]
既然可以辯識木瓜的熟度，那應該也可以辨別芒果的品種。同樣都屬於「分類(classification)」的問題。

**Comment by HSU-YU-JEN** [comment]
這類技術應該是利用影像辨識，以整數0到255作為索引來定位顏色，辨識水果成熟度
